src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -1649,8 +1670,8 @@ public void postParse(int type, InputFile f) {
 		}
 
 		// Check for external subs here
-		if (f.getFile() != null && type == Format.VIDEO && configuration.isAutoloadExternalSubtitles()) {
-			FileUtil.isSubtitlesExists(f.getFile(), this);
+		if (file.getFile() != null && type == Format.VIDEO && configuration.isAutoloadExternalSubtitles()  && !thumbOnly) {
+			FileUtil.isSubtitlesExists(file.getFile(), this);
 		}
```

Comment:
This code is being reworked in #1372.

Reply:
@Nadahar OK I will wait until you finished the #1372 but it can give you some proposal.